semantic_conventions/trace/tls.yaml

```diff
@@ -0,0 +1,204 @@
+groups:
+  - id: tls
+    prefix: tls
+    type: span
+    brief: "This document defines semantic conventions for TLS/SSL client and server Spans."
+    attributes:
+      - id: version
```

Comment:
should it be `tls.protocol.version` ? reasoning in the comment on `tls.version_protocol`

Reply:
same here.

Reply:
applied